fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/api/resource/package-info.java

```diff
@@ -17,37 +17,38 @@
 /**
  * The Resource Loader, version 0.
  *
- * <p><h3>Quick note about vocabulary in Minecraft:</h3></p>
+ * <h3>Quick note about vocabulary in Minecraft:</h3>
+ *
  * <p>Resource Pack refers to both client-sided resource pack and data pack.</p>
```

Comment:
Should the end tag be here either?

Reply:
No, the whole tag isnt needed at all. Removed.